gadget/gadget_emmc_test.go

```diff
@@ -0,0 +1,387 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) Canonical Ltd
```

Comment:
We do not put the year here anymore? Is this a new policy?

Reply:
From a legal standponit we don't have to, we should slowly migrate to year-less headers, i wanted to bring this up at the sprint but forgot